src/components/BookList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+function BookList() {
+  return (
+    <div className="List-container">
+      <div className="BookDetils">
+        <p className="Genre">Category</p>
+        <p className="BookTitle">Title</p>
+        <p className="BookAuthor">Author</p>
+        <button type="button" className="btn">Comment</button>
+        <button type="button" className="btn">Remove</button>
+        <button type="button" className="btn">Edit</button>
+      </div>
+      <div className="Scale">
+        <div className="Graph">
+          <div className="progressBar" style={{ width: 50, height: 50 }}>
+            <CircularProgressbar value={67} />
+            <p className="percentage">67%</p>
+          </div>
+          <p className="completed">Completed</p>
+        </div>
+      </div>
+      <div className="Progress">
+        <p>Current Chapter</p>
+        <p>Chapter 17</p>
+        <button type="button" className="progressBtn btn">Update Book</button>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] There should be a component representing a **single** book. The component should take in book data as a prop and render the **`title`** and **`author`**. If this component is intended for that then kindly refactor accordingly.

Reply:
Change made